commit 4212272b3889f2df155d2aa8a1a5305fe7a7d3a5 (refs/changes/10/510/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-09-28T20:41:02+00:00 (3 years ago)
    
    joyent/node-sshpk#15 Bump tweetnacl
    Reviewed by: Cody Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index db9bfc1..873a226 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sshpk",
-  "version": "1.10.0",
+  "version": "1.10.1",
   "description": "A library for finding and using SSH public keys",
   "main": "lib/index.js",
   "scripts": {
@@ -46,7 +46,7 @@
   },
   "optionalDependencies": {
     "jsbn": "~0.1.0",
-    "tweetnacl": "~0.13.0",
+    "tweetnacl": "~0.14.0",
     "jodid25519": "^1.0.0",
     "ecc-jsbn": "~0.1.1",
     "bcrypt-pbkdf": "^1.0.0"
